src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -74,11 +71,28 @@ public boolean isEnabled(final String toggleName, final boolean defaultSetting)
     }
 
     @Override
-    public boolean isEnabled(final String toggleName, final UnleashContext context ,final boolean defaultSetting) {
+    public boolean isEnabled(final String toggleName, final UnleashContext context, final boolean defaultSetting) {
+        return isEnabled(toggleName, context, defaultSetting, null);
+    }
+
+    public boolean isEnabled(final String toggleName, final BiFunction<String, UnleashContext, Boolean> fallbackAction) {
+        return isEnabled(toggleName, false, fallbackAction);
+    }
+
+    public boolean isEnabled(final String toggleName, boolean defaultSetting, final BiFunction<String, UnleashContext, Boolean> fallbackAction) {
+        return isEnabled(toggleName, contextProvider.getContext(), defaultSetting, fallbackAction);
+    }
+
+    public boolean isEnabled(final String toggleName, UnleashContext context, boolean defaultSetting, final BiFunction<String, UnleashContext, Boolean> fallbackAction) {
         FeatureToggle featureToggle = toggleRepository.getToggle(toggleName);
         boolean enabled = isEnabled(featureToggle, context, defaultSetting);
```

Comment:
I believe it would be better if we modify the private `isEnabled`-method (line 99) to take `FallbackAction` as last input and have all public `isEnabled`-methods call this one. There are some reasons for this:

- Avoid duplication of logic. 
- ensure that FallbackAction gets the enhanced context injected. 

This change would require existing `isEnabled`-methods to also send in a fallback function, this can be achieved via a lambda that just returns the fallbackValue injected to the method (or false)